install/ansible/install.sh

```diff
@@ -131,13 +162,15 @@ if [ "$service_vip" == "" ]; then
 	service_vip=$netmaster
 fi
 
-if [ "$cluster_store" == "" ]; then
-	cluster_store="etcd://localhost:2379"
+if [ "$cluster_store" = "" ]; then
+    cluster_store_type="etcd"
+	cluster_store_urls="http://localhost:2379"
```

Comment:
random spacing issue

Reply:
i'll submit a new patch to format the codes